packages/edit-site/src/components/posts-app/posts-list.js

```diff
@@ -207,6 +217,46 @@ function getItemId( item ) {
 	return item.id.toString();
 }
 
+function PostAuthorField( { item, viewType } ) {
+	const { text, icon, imageUrl } = useSelect(
+		( select ) => {
+			const { getUser } = select( coreStore );
+			const user = getUser( item.author );
+			return {
+				icon: authorIcon,
+				imageUrl: user?.avatar_urls?.[ 48 ],
+				text: user?.name,
+			};
+		},
+		[ item ]
+	);
+	const withIcon = viewType !== LAYOUT_LIST;
```

Comment:
Nit: let's rename to `withAvatar` for more clarity. Additionally and also nit is to combine the checks like ` {withAvatar && ( imageUrl ? ... : ) }